docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -82,90 +82,37 @@ If successful, your robot will show that it's **Live**.
 
 For more `micro-rdk-installer` CLI usage options, see [GitHub](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
 
-### Configure an esp32 board
-
-{{< alert title="Info" color="info" >}}
-
-The`esp32` [board](/components/board/) model is not currently provided for you as a built-in option in [the Viam app](https://app.viam.com), so you cannot use the **Config Builder** to configure this board.
-
-{{< /alert >}}
-
-To add an `esp32` board, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com) and select **Raw JSON** mode.
-
-Copy the following JSON template and paste it into your configuration inside the `"components"` array:
-
-{{< tabs name="Configure an esp32 Board" >}}
-{{% tab name="JSON Template"%}}
-
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "attributes": {
-    "pins": [
-      <int>
-    ],
-    "analogs": [
-      {
-        "pin": "<number>",
-        "name": "<your-analog-name>"
-      }
-    ]
-  },
-  "depends_on": [],
-  "model": "esp32",
-  "name": "<your-board-name>",
-  "type": "board"
-}
-```
+### Configure your robot with an esp32
```

Comment:
```suggestion
### Configure your robot with an esp32
```
```suggestion
### Configure your robot with an ESP32
```
[nit] I don't think users will realize that this is meant to match the model name string or whatnot; I think it should just be consistent with what's above on this page.

Reply:
Ok, done!